src/common/BaseAnnotator.ts

```diff
@@ -121,6 +121,15 @@ export default class BaseAnnotator {
 
     setVisibility = (visibility: boolean): void => {
         this.store.dispatch(store.setVisibilityAction(visibility));
+
+        if (!this.rootEl) {
+            return;
+        }
+        if (visibility) {
```

Comment:
Should this be handled by the parent at this point?

Reply:
It seems like we're using redux to store the state, but not relying on it to hide and show the annotations, but rather rely on the value passed into `setVisibility`. What are your thoughts on having `RegionContainer` connect to this state and have `RegionAnnotations` use that as a basis to render `RegionList`?

Reply:
> It seems like we're using redux to store the state, but not relying on it to hide and show the annotations, but rather rely on the value passed into `setVisibility`. What are your thoughts on having `RegionContainer` connect to this state and have `RegionAnnotations` use that as a basis to render `RegionList`?

Jared thinks this logic should live in the higher level. Otherwise we'll have to replicate this logic for all future annotation types.